src/storage/class-sidebar.php

```diff
@@ -0,0 +1,334 @@
+<?php
+/**
+ * Sidebar_Storage class file
+ *
+ * @package wp-widget-control
+ */
+
+namespace Alley\WP\Widget_Control\Storage;
```

Comment:
Semantically, why `Storage`? 

Can you use a sidebar or a widget without being stored in the database?